CONTRIBUTING.md

```diff
@@ -82,3 +90,55 @@ export class MyConstruct extends Construct {
   * Other import methods normally require access to the context API which is beyond the scope of these constructs. We'd like to address this in the future.
 * Constructs should implement an interface so that the import methods (e.g. `.fromSomethingAttributes()`) can return the interface type.
 
+
+
+## Testing
+
+Tests are fundamental to building good constructs. The following testing guidelines should be followed:
+
+* All tests should be stored in the `test/aws-<something>` subdirectory. For example, if you are creating
+a construct for the AppStream service, then the module name would be `aws-appstream` and tests would go in the `/test/aws-appstream` directory.
+* Fine-grained tests should be implemented for all constructs with a goal of 100% coverage. For example,
+if you were creating an Application construct in the AppSteam module, your tests should be in the `test/aws-appstream/Application.test.ts` file.
+* Integration tests should be written for all constructs. For example, if you are creating an integration test
+for the Application construct in the AppStream module, your tests should be in the `test/aws-appstream/Application.integ.ts` file.
```

Comment:
I think the file name should be lower case.

```suggestion
* Fine-grained tests should be implemented for all constructs with a goal of 100% coverage. For example,
if you were creating an Application construct in the AppSteam module, your tests should be in the `test/aws-appstream/application.test.ts` file.
* Integration tests should be written for all constructs. For example, if you are creating an integration test
for the Application construct in the AppStream module, your tests should be in the `test/aws-appstream/application.integ.ts` file.
```

Reply:
Fair, I was still working off of the Typescript convention and not the CDK one. Will fix.